### ISSUE TITLE
unknown method GetSessionRecordingConfig for service proto.AuthService

### ISSUE THREAD

**Issue by benarent** [gravitational/teleport#7542]
### Description

**What happened**:

I tried 7.0.0 on a new Mac setup and I had the below error. 

```
admin@Admins-iMac teleport % sudo teleport start \
   --roles=node \
   --token=xxxx \
   --ca-pin=sha256:ef56e342d09590d991b9250357bc5d11b836a2d2f7908ec84256c55446057b23 \
   --auth-server=acme-demo.teleport.sh:443
INFO [PROC:1]    Joining the cluster with a secure token. service/connect.go:355
INFO [PROC:1]    Node has obtained credentials to connect to the cluster. service/connect.go:384
INFO [PROC:1]    The process successfully wrote the credentials and state of Node to the disk. service/connect.go:425
INFO [PROC:1]    Node: features loaded from auth server: Kubernetes:true App:true DB:true OIDC:true SAML:true AccessControls:true AdvancedAccessWorkflows:true Cloud:true  service/connect.go:60
WARN [PROC:1]    Teleport process has exited with error. error:[
ERROR REPORT:
Original Error: *trace.NotImplementedError unknown method GetSessionRecordingConfig for service proto.AuthService
Stack Trace:
	/tmp/build-darwin-amd64/go/src/github.com/gravitational/teleport/vendor/github.com/gravitational/teleport/api/client/client.go:1455 github.com/gravitational/teleport/api/client.(*Client).GetSessionRecordingConfig
	/tmp/build-darwin-amd64/go/src/github.com/gravitational/teleport/lib/auth/clt.go:1744 github.com/gravitational/teleport/lib/auth.(*Client).GetSessionRecordingConfig
	/tmp/build-darwin-amd64/go/src/github.com/gravitational/teleport/lib/service/service.go:1664 github.com/gravitational/teleport/lib/service.(*TeleportProcess).initSSH.func1
	/tmp/build-darwin-amd64/go/src/github.com/gravitational/teleport/lib/service/supervisor.go:457 github.com/gravitational/teleport/lib/service.(*LocalService).Serve
	/tmp/build-darwin-amd64/go/src/github.com/gravitational/teleport/lib/service/supervisor.go:246 github.com/gravitational/teleport/lib/service.(*LocalSupervisor).serve.func1
	/usr/local/go/src/runtime/asm_amd64.s:1371 runtime.goexit
User Message: unknown method GetSessionRecordingConfig for service proto.AuthService] service:ssh.node service/supervisor.go:251
ERRO [PROC:1]    Critical service ssh.node has exited with error unknown method GetSessionRecordingConfig for service proto.AuthService, aborting. service/signals.go:144
INFO [NODE:1]    Shutting down immediately. service/service.go:1848
INFO [NODE:1]    Exited. service/service.go:1871
ERROR: unknown method GetSessionRecordingConfig for service proto.AuthService

```

**What you expected to happen**:

### Reproduction Steps
As minimally and precisely as possible, describe step-by-step how to reproduce the problem.
1. Try to connect a Mac to Teleport
2. `teleport start --roles=node`

### Server Details
- Teleport version (run `teleport version`): 7.0.0
- Server OS (e.g. from `/etc/os-release`): Darwin

**Comment by benarent** [comment]
Note: This issue is due to stricter following for Teleport Version as outlined in RFD 12 - Teleport versioning.  

TLDR: Always update Auth+Proxy before upgrading nodes.
